modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -20,10 +19,27 @@ def index
           render json: claims
         end
 
+        def show
+          claim = ClaimsApi::AutoEstablishedClaim.get_by_id_or_evss_id(params[:id])
+
+          if claim.present?
+            render json: { id: claim[:id], type: claim[:claim_type] }
```

Comment:
AC in the ticket is to make sure that the id we pass back is the one the consumer passed in.
easiest way to do this would be to just send back params[:id] here instead

Reply:
updated 👍